include/Graph.h

```diff
@@ -48,14 +48,10 @@ class LMMS_EXPORT Graph : public QWidget, public ModelView
 		LinearStyle, //!< connect each 2 samples with a line, with wrapping
 		LinearNonCyclicStyle, //!< LinearStyle without wrapping
 		BarStyle, //!< draw thick bars
-		NumGraphStyles
+		NumGraphStyles,
+		BarCenterGradStyle //!< draw color gradient coming from center
 	};
 
-	/**
-	 * @brief Constructor
-	 * @param _width Pixel width of widget
-	 * @param _height Pixel height of widget
-	 */
```

Comment:
Why did you remove these comments?

Reply:
Whoa... um, better question, why do I have a version of LMMS RC6 on here in a folder labeled as RC8... that's 100% my bad, I copied Graph.h to get the original file back then made the change needed, but apparently the copy of LMMS I copied it from was RC6.  That's the second time I made a dumb mistake like this, I think I need to go through all my LMMS copies and make sure they're in the correct spots... I'll fix that right away, I can't believe I didn't catch that.

Reply:
Oooooh, I copied Graph.h from LMMS on Github to get it back to the original, but I accidentally copied it from stable1.2 rather than master.  My bad, sorry about that.